src/functions/env.ts

```diff
@@ -45,8 +45,8 @@ const LINE_RE = new RegExp(
   "(\\w+)" +               //   key
   "\\s*=\\s*" +            //   separator (=)
   "(" +                    //   begin optional value
-  "\\s*'(?:\\'|[^'])*'|" + //     single quoted or
-  '\\s*"(?:\\"|[^"])*"|' + //     double quoted or
+  "\\s*'(?:\\\\'|[^'])*'|" + //     single quoted or
```

Comment:
Customers are expected to escape quotes with double slashes?

Reply:
Ah this was actually fixing a bug in the existing implementation that I found while writing the export script. I'm going to pull this change as another PR instead of earmarking it into this one.

Reply:
Merged as https://github.com/firebase/firebase-tools/pull/3703.